imports/plugins/core/checkout/client/components/cartIcon.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component, PropTypes } from "react";
+import { Reaction } from "/client/api";
+
+class CartIcon extends Component {
+  static propTypes = {
+    cart: PropTypes.object
+  }
+
+  handleClick = (event) => {
+    event.preventDefault();
+    return $("#cart-drawer-container").fadeOut(300, function () {
```

Comment:
jQuery will break SSR support, + its very un-react-like. Try with VelocityJS

Example in our code:

https://github.com/reactioncommerce/reaction/blob/master/imports/plugins/core/dashboard/client/components/actionView.js#L392

```js
import { VelocityTransitionGroup } from "velocity-react";

  get enterAnimation() {
    return {
      animation: {
        display: "flex",
        position: "absolute",
        left: 20,
        opaticy: 1
      },
      duration: 200
    };
  }

  get leaveAnimaton() {
    return {
      animation: {
        display: "flex",
        position: "absolute",
        left: -30,
        opaticy: 0
      },
      duration: 200

    };
  }

            <VelocityTransitionGroup
              enter={this.enterAnimation}
              leave={this.leaveAnimaton}
            ></VelocityTransitionGroup>
```

---

Alternatively, add a `// TODO: Remove jQuery`  and wrap any jquery `$()` selector in an if like so.

```js
// TODO: Remove jQuery
if (window && window.jQuery) {
  $("#cart-drawer-container").fadeOut(300.....
}
```